spec/utility/entity_merger_spec.rb

```diff
@@ -1,61 +1,846 @@
 require 'rails_helper'
 
-describe 'Merging Entities' do
- 
-  # def entity_merge(source:, dest:)
-  # end
+describe 'Merging Entities', :merging_helper do
```

Comment:
i trust that the ~900 lines of test code below work as their doc strings advertise and have not inspected them overly closely. i commend you for writing them with such attention to detail! <3